third_party/jsign/index.mjs

```diff
@@ -15,9 +15,9 @@
 import {spawn} from 'node:child_process';
 import {resolve} from 'node:path';
 
-import {downloadHttpsFile} from '../../src/build/download_file.mjs';
-import {getFileChecksum} from '../../src/build/get_file_checksum.mjs';
-import {getRootDir} from '../../src/build/get_root_dir.mjs';
+import {downloadHttpsFile} from '../../apps/client/build/download_file.mjs';
```

Comment:
Infrastructure code must not depend on apps code. Move those to a shared infrastructure folder.